airflow/providers/docker/CHANGELOG.rst

```diff
@@ -19,6 +19,23 @@
 Changelog
 ---------
 
+2.2.0
+.....
+
+Features
+~~~~~~~~
+
+* ``Add a Docker Taskflow decorator (#15330)``
```

Comment:
This change requires Aiirflow 2.2 which is yet to be released, so we probably shouldn't be releasing this packages yett.

Reply:
Quite the opposite - we should release them so that we can release Airflow 2.2 with this feature working. We've done that in the past - we need to release provider's first and Airflow after.

Reply:
The decorator is not functional in Airflow 2.1, but this is no harm to have it added.

Reply:
Mailing list discussion: https://lists.apache.org/thread.html/rbe2552f5c49114378a9aedb7ceee906976158d38b44e955fed18000f%40%3Cdev.airflow.apache.org%3E